monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinition.java

```diff
@@ -43,7 +43,8 @@
     private boolean _distributedExecution;
     private String _dateForOneTimeSchedule;
 	private Map<String,String> _jobMetadataProperties;
-    private boolean _runOnHadoop; 
+    private boolean _runOnHadoop;
+    private String _hotFolder;
```

Comment:
Oh. The beauty of not printing whitespaces (didn't Github used to highlight the actual change in a darker color?)